apps/ui/src/views/Space/Settings.vue

```diff
@@ -240,8 +239,6 @@ function handleSettingsSave() {
 }
 
 function handleControllerSave(value: string) {
-  changeControllerModalOpen.value = false;
-
```

Comment:
We no longer close modal on save, so loading state is not visible:
<img width="1500" alt="image" src="https://github.com/user-attachments/assets/318e20e2-470c-4637-9710-ce372dae8248" />